packages/nouns-contracts/contracts/governance/NounsDAOExecutorV2.sol

```diff
@@ -0,0 +1,52 @@
+// SPDX-License-Identifier: BSD-3-Clause
+
+/// @title The Nouns DAO executor and treasury
+
+/*********************************
+ * ░░░░░░░░░░░░░░░░░░░░░░░░░░░░░ *
+ * ░░░░░░░░░░░░░░░░░░░░░░░░░░░░░ *
+ * ░░░░░░█████████░░█████████░░░ *
+ * ░░░░░░██░░░████░░██░░░████░░░ *
+ * ░░██████░░░████████░░░████░░░ *
+ * ░░██░░██░░░████░░██░░░████░░░ *
+ * ░░██░░██░░░████░░██░░░████░░░ *
+ * ░░░░░░█████████░░█████████░░░ *
+ * ░░░░░░░░░░░░░░░░░░░░░░░░░░░░░ *
+ * ░░░░░░░░░░░░░░░░░░░░░░░░░░░░░ *
+ *********************************/
+
+// LICENSE
+// NounsDAOExecutor.sol is a modified version of Compound Lab's Timelock.sol:
+// https://github.com/compound-finance/compound-protocol/blob/20abad28055a2f91df48a90f8bb6009279a4cb35/contracts/Timelock.sol
+//
+// Timelock.sol source code Copyright 2020 Compound Labs, Inc. licensed under the BSD-3-Clause license.
+// With modifications by Nounders DAO.
+//
+// Additional conditions of BSD-3-Clause can be found here: https://opensource.org/licenses/BSD-3-Clause
+//
+// MODIFICATIONS
+// NounsDAOExecutor.sol modifies Timelock to use Solidity 0.8.x receive(), fallback(), and built-in over/underflow protection
+// This contract acts as executor of Nouns DAO governance and its treasury, so it has been modified to accept ETH.
+
+pragma solidity ^0.8.6;
+
+import { NounsDAOExecutor } from './NounsDAOExecutor.sol';
+
+contract NounsDAOExecutorV2 is NounsDAOExecutor {
+    
+    error RedeemFailed();
+
+    constructor(address admin_, uint256 delay_) NounsDAOExecutor(admin_, delay_) {}
+
+    function redeem(address account, uint256 amount) external {
+        require(msg.sender == admin, 'Only admin');
+
+        (bool success, ) = account.call{ value: amount, gas: 30_000 }(new bytes(0));
```

Comment:
you probably thought about this, just capturing for later: we probably want to do a "safe send" that uses WETH if sending ETH fails

Reply:
either that, or we could use an escrow contract that the ragequitter can "pull" ETH from